operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -69,7 +69,7 @@ An authentication token is required to access the API gateway and to use the `sa
     1. Find the required `templateUuid` value with BOS.
 
        ```bash
-       ncn# cray bos session describe XNAME | grep templateUuid
+       ncn# cray bos session describe boss_session | grep templateUuid
```

Comment:
On any ncn run:
```suggestion
     cray bos session describe boss_session | grep templateUuid
```

Reply:
The XNAME above is incorrect. "boss_session" should perhaps be bos_session

Reply:
Fixed above corrections.